tests/conftest.py

```diff
@@ -64,7 +107,8 @@ def environ(
     data_dir,
     config_dir,
     runtime_dir,
-    root_dir
+    root_dir,
+    asyncio_patch
```

Comment:
I don't think we need to use this fixture in the environ. It will be picked up by the serverapp fixture.

Reply:
yep with my last commit it should not be needed here.